### PR TITLE
Added Tests For Endpoint Service

### PR DESCRIPTION
**Pull request by zanpizmoht**
Added tests for creating, deleting and editing Endpoints.

Signed-off-by: zanpizmoht <zan.pizmoht@comtrade.com>

In this PR new test scenarios for Endpoint service are created.

**Related Issue**
_None_

**Description of the solution adopted**
Test scenarios that are added in this PR are checking functionalities regarding Endpoints. All scenarios are added to `EndpointServiceI9n.feature`. Scenarios are testing if Endpoints are correctly created, edited or deleted. There are also two scenarios added for testing Endpoint in different accounts.

**Screenshots**
/

### CODE REVIEW

**Comment by Coduz** [comment]
CQ submitted: [CQ 21396](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21396)

**Review by Coduz — APPROVED** [review]

**Review by LeoNerdoG — COMMENTED** [review]
Hi @zanpizmoht,
I have reviewed the PR and there are some changes that need to be made. 
Please take a look at the comments and fix the selected lines.

**Review by LeoNerdoG — COMMENTED** [review]
Hi @zanpizmoht,
I have reviewed the PR and there are some changes that need to be made. 
Please take a look at the comments and fix the selected lines.

File: qa/integration/src/test/resources/features/endpoint/EndpointServiceI9n.feature [hunk 1]
```diff
@@ -0,0 +1,935 @@
...
+  Exception should be raised
+    Given I login as user with name "kapua-sys" and password "kapua-password"
+    Given I expect the exception "KapuaIllegalArgumentException" with the text "*"
+    And I create endpoint with schema "http://abc.com", domain "abc.com" and port 2222
+    Then An exception was thrown
+    And I logout
+
+  Scenario: Creating Endpoint With Schema Containing "https://"
+  Login as kapua-sys
+  Create a endpoint with Schema containing "http://"
```
Comment (LeoNerdoG) [inline qa/integration/src/test/resources/features/endpoint/EndpointServiceI9n.feature]: This should be changed to "https://"!

**Review by LeoNerdoG — CHANGES_REQUESTED** [review]
Hey @zanpizmoht,
I have checked this PR again and there are still some comments that need to be addessed. Please review them and fix them accordingly.

**Review by LeoNerdoG — APPROVED** [review]
Everything is OK now from QA side. 
@Coduz can you please check the status of CQ and rerun failing tests so we will have accurate codeCov? 

thanks!

**Comment by Coduz** [comment]
[CQ approved](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21396#c5) 🎉

**Pull request merged**